storio-common-annotations-processor/build.gradle

```diff
@@ -1,8 +1,22 @@
 apply plugin: 'java'
+apply plugin: 'jacoco'
```

Comment:
Maybe we can extract this as separate `.gradle` script and use it in the same manner as `gradle/jacoco-android.gradle`